README.md

```diff
@@ -62,7 +50,7 @@ Application:
 
 ## Available commands
 
-### download
+### get
 This command will fetch all the variables from the defined sources and put them in proper file(s).
```

Comment:
hmm, and the code below says it "Fetches configuration for configured sources and outputs it on stdout"